benchmark_list.cmake

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2022, Nataliya Korovkina <malus.brandywine@gmail.com>
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+# Set the list of benchmark applications to be included into the image
+# Uncomment command set() when needed
+
+# Application to benchmark seL4 IPC
+# default AppIpcBench is ON
```

Comment:
could be a bit less verbose/redundant everywhere
```suggestion
# default is ON
```

Reply:
Agreed, I'll change it